cloud/azure/bin/lib/azure_log.sh

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env bash
+
+readonly AZURE_LOG_STORAGE_ACCOUNT_NAME="civiformdeploymentlog"
```

Comment:
question (blocking): should we have this be read in from config, since account names have to be globally unique?

Reply:
Ah dang, yeah I didn't realize the storage account name had to be globally unique

Reply:
removed